pyproject.toml

```diff
@@ -20,6 +20,7 @@ keywords = ['exasol', 'udf', 'script-languages']
 python = ">=3.8,<4"
 importlib_metadata = ">=4.6.0"
 importlib-resources = ">=5.4.0"
+networkx = "2.8.2"
```

Comment:
```suggestion
networkx = "2.8.2" # We pinned networkx to this version, because in newer versions it throws an exception, see https://github.com/exasol/integration-test-docker-environment/issues/228
```